tests/test_event.py

```diff
@@ -99,6 +101,46 @@
 ]
 
 
+# -------------------------------------------------
+# fixtures
+# -------------------------------------------------
+
+
+@pytest.fixture
+def new_event(session, new_user):
+    event = add_new_event(TestApp.event_test_data, session)
+    return event
+
+
+@pytest.fixture
+def new_user(session):
```

Comment:
maybe you can use existing fixtures?

Reply:
Tried but didn't really find a proper user fixture inside the file. ill try going over the code again

Reply:
There are enough user fixtures, use one of the others.